### DISCUSSION TITLE
一些博客排版优化小 trick

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#33] [Comments]
# 一些博客排版优化小 trick

我的博客好像还用了不少奇怪的小 trick 来优化 （hopefully） 排版，在这整理一下，分享出来，也是怕几年（几天）后想不起来这些奇怪的东西是在干什么。
主要是 text justify 相关和标点挤压相关。

https://ouuan.moe/post/2023/02/typography-tricks

**Comment by ouuan** [comment]
每次都像这样文章内容和博客样式紧密耦合，感觉以后改样式甚至换博客会非常艰难（

**Comment by ouuan** [comment]
前两天终于把咕了一年多的 MIUI 13 更新了，发现系统字体换成了 MiSans，结果标点挤压没了，于是换回了小米兰亭 Pro，不知道当年为小米兰亭 Pro 添加 chws 特性的设计师如果看到了会怎么想（
